packages/lib/commands/synchronize.ts

```diff
@@ -35,16 +35,20 @@ export const runtime = (): CommandRuntime => {
 					return 'auth';
 				}
 
-				reg.logger().info('Not authentified with sync target - please check your credential.');
+				reg.logger().error('Not authentified with sync target - please check your credential.');
```

Comment:
```suggestion
				reg.logger().error('Not authenticated with sync target - please check your credentials.');
```

This sentence has 2 errors qnd since i is not a translatable string, we should fix it asap:
- `authentified` is not a word. It does not exist.
- the word credential must be used as a plural here.